install/kots/manifests/kots-config.yaml

```diff
@@ -379,6 +379,13 @@ spec:
           default: "0"
           help_text: Enables additional customization options. Enable only when you know what you are doing!
 
+        - name: customization_patch
+          title: Gitpod customization patch (YAML file)
+          type: file
+          required: false
+          when: '{{repl ConfigOptionEquals "advanced_mode_enabled" "1" }}'
+          help_text: A file configured to override annotations, labels and environment variables in components. See our [advanced documentation](https://www.gitpod.io/docs/self-hosted/latest/advanced/customization) for more details and the format.
```

Comment:
Can we have a statement here that changing this value lets the installation fail after the initial (first) installation?

Reply:
We can, and I will do that.

Currently, the `help_text` isn't actually displayed - see https://github.com/replicated-collab/replicated-gitpod/issues/29 which I raised this morning